src/resultsdisassemblypage.ui

```diff
@@ -87,7 +87,7 @@
         </property>
         <property name="icon">
          <iconset theme="go-previous">
-          <normaloff>.</normaloff>.</iconset>
+          <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
```

Comment:
unrelated changes?

Reply:
huh? Yes no idea what QDesigner is doing

Reply:
maybe it is because I pressed restore changes after it crashed